packages/docusaurus-utils-validation/src/__tests__/validationSchemas.test.ts

```diff
@@ -106,4 +107,17 @@ describe('validation schemas', () => {
   test('RehypePluginsSchema', () => {
     testMarkdownPluginSchemas(RehypePluginsSchema);
   });
+
+  const useTest = <T>(schema: Joi.Schema) => (value: T) =>
+    Joi.attempt(value, schema);
+
+  test('URISchema', () => {
+    const validURL = 'https://docusaurus.io';
+    const doubleHash = 'https://docusaurus.io#github#/:';
+    const invalidURL = 'https://docusaurus.io?search=  ';
+    const testURL = useTest(URISchema);
+    expect(testURL(validURL)).toBe(validURL);
+    expect(testURL(doubleHash)).toBe(doubleHash);
+    expect(() => testURL(invalidURL)).toThrowErrorMatchingSnapshot();
```

Comment:
Why not using testOk/testFail like the other tests?

Also we should have a test with the exact URL of the bug report, to ensure this specific bug is fixed

Reply:
ok